pyht/client.py

```diff
@@ -45,13 +44,24 @@
     ]
 
 
+class Format(Enum):
```

Comment:
I added this wrapper enum because protobuf seems to have changed how the enum type wrapper classes look in the generated code; the `api_pb2.Format.FORMAT_X` access pattern no longer works with the syntax highlighter or pyright due to this (although it does work at runtime). Since we expose the generated Format enum in __init__.py and it may be used by client code, it's necessary to keep doing so.

Reply:
LGTM :+1